api/src/bigquerytest/java/org/pmiops/workbench/api/CohortBuilderControllerBQTest.java

```diff
@@ -898,23 +922,23 @@ public void validateAttribute() {
         createSearchRequests(
             Domain.CONDITION.toString(), ImmutableList.of(demo), new ArrayList<>());
     try {
-      controller.countParticipants(cdrVersion.getCdrVersionId(), searchRequest);
+      controller.countParticipants(WORKSPACE_NAMESPACE, WORKSPACE_ID, searchRequest);
       fail("Should have thrown BadRequestException!");
     } catch (BadRequestException bre) {
       assertThat(bre.getMessage()).isEqualTo("Bad Request: attribute operator null is not valid.");
     }
```

Comment:
Should we convert all these test to use `@Test(expected = BadRequestException.class)`